examples/window-app/silabs/src/WindowManager.cpp

```diff
@@ -725,7 +725,7 @@ void WindowManager::UpdateLCD()
 #if CHIP_ENABLE_OPENTHREAD
     if (mState.isThreadProvisioned)
 #else
-    if (mState.isWiFiProvisioned)
+    if (ConnectivityMgr().IsWiFiStationProvisioned())
```

Comment:
Same comment here

Reply:
Replaced this to get provision status from BaseApplication.cpp